src/core/agent/services/identifierService.ts

```diff
@@ -410,15 +417,45 @@ class IdentifierService extends AgentService {
           isPending,
         });
 
+        this.props.eventEmitter.emit<IdentifierStateChangedEvent>({
```

Comment:
Can we remove these changes? I would prefer if we don't "start" the app until the recovery is done. So `AppWrapper` will get get all identifiers etc as normal AFTER we have finished recovering.

We can show something in the UI that a recovery is in process